src/analysis/mean_inequalities.lean

```diff
@@ -755,6 +755,52 @@ begin
     hg_zero,
 end
 
+lemma lintegral_rpow_add_lt_top_of_lintegral_rpow_lt_top {p : ℝ}
+  {f g : α → ennreal} (hf : measurable f) (hf_top : ∫⁻ a, (f a) ^ p ∂μ < ⊤)
+  (hg : measurable g) (hg_top : ∫⁻ a, (g a) ^ p ∂μ < ⊤) (hp1 : 1 ≤ p) :
+  ∫⁻ a, ((f + g) a) ^ p ∂μ < ⊤ :=
+begin
+  have hp0_lt : 0 < p, from lt_of_lt_of_le zero_lt_one hp1,
+  have hp0 : 0 ≤ p, from le_of_lt hp0_lt,
+  have hp_sub_one_nonneg : 0 ≤ p - 1, by simp [hp1],
```

Comment:
```suggestion
```
Since you're only using this once, you can remove it from here, and replace its only use below with `(by simp [hp1])`.